grant/grant.go

```diff
@@ -51,21 +53,70 @@ var (
 	claimedGrantsCounter         = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "claimed_grants_total",
-			Help: "Number of claimed grants.",
+			Help: "Number of grants claimed since start.",
 		},
 		[]string{},
 	)
 	redeemedGrantsCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "redeemed_grants_total",
-			Help: "Number of redeemed grants.",
+			Help: "Number of grants redeemed since start.",
 		},
 		[]string{"promotionId"},
 	)
 )
 
+type grantService struct {
+	pool                      *redis.Pool
+	outstandingGrantCountDesc *prometheus.Desc
+	completedGrantCountDesc   *prometheus.Desc
+}
+
+// Describe comment
+func (gs *grantService) Describe(ch chan<- *prometheus.Desc) {
+	ch <- gs.outstandingGrantCountDesc
+	ch <- gs.completedGrantCountDesc
+}
+
+// Collect comment
+func (gs *grantService) Collect(ch chan<- prometheus.Metric) {
+	conn := gs.pool.Get()
```

Comment:
oops - upon self-review realized that I missed a `defer conn.Close()` here
will push up fix